orchagent/dash/dashvnetorch.cpp

```diff
@@ -430,13 +430,18 @@ bool DashVnetOrch::addVnetMap(const string& key, VnetMapBulkContext& ctxt)
             SWSS_LOG_INFO("Not creating VNET map for %s since VNET %s doesn't exist", key.c_str(), ctxt.vnet_name.c_str());
             return false;
         }
-        return addOutboundCaToPa(key, ctxt) && addPaValidation(key, ctxt);
+
+        remove_from_consumer = addOutboundCaToPa(key, ctxt);
```

Comment:
Could you add a comment here that if addOutboundCaToPa fails, we intend to skip addPaValidation  and a TODO to perhaps handle addPaValidation failure?

Reply:
Will add a note. AddPaValidation pre-bulk doesn't really have a way to fail since it's just adding items to the bulker, is the TODO still needed?

Reply:
Ok, then we can skip the TODO.